src/lib/entry-point-transforms.ts

```diff
@@ -59,19 +64,35 @@ export const transformSources =
     // 3. FESM15: ROLLUP
     log.info('Bundling to FESM15');
     const fesm15File = path.resolve(artefacts.stageDir, 'esm2015', entryPoint.flatModuleFile + '.js');
+
     await rollup({
       moduleName: entryPoint.moduleId,
       entry: tsOutput.js,
       format: 'es',
       dest: fesm15File,
-      externals: entryPoint.externals
+      externals: {
+        ...entryPoint.externals,
+        [TS_HELPERS_LIB.name]: TS_HELPERS_LIB.name
+      }
     });
     await remapSourceMap(fesm15File);
 
     // 4. FESM5: TSC
     log.info('Bundling to FESM5');
     const fesm5File = path.resolve(artefacts.stageDir, 'esm5', entryPoint.flatModuleFile + '.js');
-    await downlevelWithTsc(fesm15File, fesm5File);
+    const downLevelTmp = path.resolve(pkg.workingDirectory, entryPoint.flatModuleFile + '.js');
+    await downlevelWithTsc(fesm15File, downLevelTmp);
```

Comment:
this is required because without rollup typescript helpers are not cherry picked.

ie it will be;
```
import * as tslib1 from 'tslib';
```

rather than
```ts
import { __extends } from 'tslib';
```